eng/common-variables.yml

```diff
@@ -12,7 +12,13 @@ variables:
     value: ''
   - name: _InternalPublishArg
     value: ''
+  - name: PostBuildSign
+    value: true
```

Comment:
this is hardcoded to true right now so we don't fail the signing approval right?

Reply:
I got the definition approved for signing, but it's hardcoded off for now until I get a full stack built and verified.

This in-build signed build succeeded: https://dev.azure.com/dnceng/internal/_build/results?buildId=1925851&view=results